docs/getting_started/faq.md

```diff
@@ -77,7 +77,7 @@ If your software requires access to specific hardware or operating system servic
 ## Is there detailed documentation for the REST APIs provided by the components in {{site.data.keyword.ieam}}?
 {: #rest_doc}
 
-Yes. For more information, see [{{site.data.keyword.ieam}} APIs](../api/edge_rest_apis.md). 
+Yes. For more information, see [{{site.data.keyword.ieam}} APIs](../api/index.md).
```

Comment:
I'm unsure of the proper way to link to index files.  I suspect that we may want to drop the "index.md" reference entirely from the URL so that we don't have two competing URLs for the same resource.

Reply:
There were two - three cases similar to this. Personally, I think `../api/` is ambiguous and would prefer to link to a real file. But also not a fan of multiple index.md files sprinkled around the directory structure.  Oh, `../api/edge_rest_api.md` isn't a file. Its a tag within index.md  Technically, my edit fixed a broken link.  Suggest we leave this commit.  We should circle back.